modules/mobile/docs/openapi.yaml

```diff
@@ -230,6 +230,126 @@ paths:
       security:
         - Bearer: []
       summary: /v0/appointments
+  /v1/appointments:
```

Comment:
@jperk51 I versioned the endpoint here but It is just adding 3 fields which could technically be done on v0. historically we've just tacked on new fields if there wasn't any schema changes to existing fields but we can go either way on it.